diff -ur rstudio-0.99.489-old/src/cpp/core/CMakeLists.txt  rstudio-0.99.489-new/src/cpp/core/CMakeLists.txt
--- rstudio-0.99.489-old/src/cpp/core/r_util/REnvironmentPosix.cpp	2015-11-04 18:53:21.000000000 +0100
+++ rstudio-0.99.489-new/src/cpp/core/r_util/REnvironmentPosix.cpp	2015-11-30 16:58:34.858119178 +0100
@@ -87,10 +87,7 @@
 {
    // define potential paths
    std::vector<std::string> rScriptPaths;
-   rScriptPaths.push_back("/usr/bin/R");
-   rScriptPaths.push_back("/usr/local/bin/R");
-   rScriptPaths.push_back("/opt/local/bin/R");
-   rScriptPaths.push_back("/Library/Frameworks/R.framework/Resources/bin/R");
+   rScriptPaths.push_back("@R@/bin/R");
    return scanForRScript(rScriptPaths, pErrMsg);
 }

@@ -228,8 +225,7 @@
       // scan in standard locations as a fallback
       std::string scanErrMsg;
       std::vector<std::string> rScriptPaths;
-      rScriptPaths.push_back("/usr/local/bin/R");
-      rScriptPaths.push_back("/usr/bin/R");
+      rScriptPaths.push_back("@R@/bin/R");
       FilePath scriptPath = scanForRScript(rScriptPaths, &scanErrMsg);
       if (scriptPath.empty())
       {
